Std/Data/List/Count.lean

```diff
@@ -173,9 +174,11 @@ theorem count_pos_iff_mem {a : α} {l : List α} : 0 < count a l ↔ a ∈ l :=
 theorem not_mem_of_count_eq_zero {a : α} {l : List α} (h : count a l = 0) : a ∉ l :=
   fun h' => Nat.ne_of_lt (count_pos_iff_mem.2 h') h.symm
 
+@[simp]
 theorem count_eq_zero {l : List α} : count a l = 0 ↔ a ∉ l :=
   ⟨not_mem_of_count_eq_zero, count_eq_zero_of_not_mem⟩
 
+@[simp]
```

Comment:
While I like `count_pos_iff_mem`, I'm not yet convinced of these two. Do they really help more than they hinder? It would be nice to see specific cases.

Reply:
I don't have specific use cases. Let's drop these two, at least for now.